src/main/app/claims/responseModelConverter.ts

```diff
@@ -422,103 +422,103 @@ export class ResponseModelConverter {
       expenses.push({
         type: ExpenseType.MORTGAGE,
         frequency: monthlyExpenses.mortgage.schedule.value as PaymentFrequency,
-        amountPaid: monthlyExpenses.mortgage.amount
+        amount: monthlyExpenses.mortgage.amount
       })
     }
 
     if (monthlyExpenses.rent && monthlyExpenses.rent.populated) {
       expenses.push({
         type: ExpenseType.RENT,
         frequency: monthlyExpenses.rent.schedule.value as PaymentFrequency,
-        amountPaid: monthlyExpenses.rent.amount
+        amount: monthlyExpenses.rent.amount
       })
     }
 
     if (monthlyExpenses.councilTax && monthlyExpenses.councilTax.populated) {
       expenses.push({
         type: ExpenseType.COUNCIL_TAX,
         frequency: monthlyExpenses.councilTax.schedule.value as PaymentFrequency,
-        amountPaid: monthlyExpenses.councilTax.amount
+        amount: monthlyExpenses.councilTax.amount
       })
     }
 
     if (monthlyExpenses.gas && monthlyExpenses.gas.populated) {
       expenses.push({
         type: ExpenseType.GAS,
         frequency: monthlyExpenses.gas.schedule.value as PaymentFrequency,
-        amountPaid: monthlyExpenses.gas.amount
+        amount: monthlyExpenses.gas.amount
       })
     }
 
     if (monthlyExpenses.electricity && monthlyExpenses.electricity.populated) {
       expenses.push({
         type: ExpenseType.ELECTRICITY,
         frequency: monthlyExpenses.electricity.schedule.value as PaymentFrequency,
-        amountPaid: monthlyExpenses.electricity.amount
+        amount: monthlyExpenses.electricity.amount
```

Comment:
same  here as well.
This will require change in claim-store to reflect the rename, otherwise response submission will fail due to 400.

One option would be don't rename here as this is converter to map.